tools/makemanifest.py

```diff
@@ -53,7 +62,7 @@ def include(manifest):
             # Applies to includes and input files.
             prev_cwd = os.getcwd()
             os.chdir(os.path.dirname(manifest))
-            exec(f.read())
+            exec(f.read(), globals(), {"args": kwargs})
```

Comment:
If it just passes kwargs directly as locals then they can be accessed without the args.  Eg:
```python
exec(f.read(), globals(), kwargs)

...

include("path.py", extra_features=True)

...

if extra_features:
    # freeze extra modules

... or to support optional args ...

if 'extra_features' in locals() and extra_features:
    # ...

... or ...

if locals().get('extra_features', False):
    # ...
```

Also, should these kwargs inherit down to sub-sub-includes?

Reply:
Yep that's exactly what I did first.

But I thought that `args` was simpler and more obvious than locals().

I actually considering going overboard and making args a type with __getattr__ with any unknown field returning False.

I'm happy either way though!

Reply:
Going "overboard" actually sounds good here, maybe called it `options` and then one does `if options.extra_features: ...`?  It's not that complicated.

Reply:
Done